pep_sphinx_extensions/pep_theme/static/style.css

```diff
@@ -121,6 +122,7 @@ pre {
 /* Contents rules */
 details > summary {
     font-weight: bold;
+    word-break: break-all;
```

Comment:
Is `details > summary` a right place?

Reply:
Wow, that's embarrassing...I meant to put it in the `pre` block right above, but wasn't paying as much attention as I should have (since I was testing it live in the browser dev tools rather than waiting for a full rebuild). My mistake; fixed now.